boards/renesas/rzg3s_smarc/doc/index.rst

```diff
@@ -0,0 +1,234 @@
+.. zephyr:board:: rzg3s_smarc
+
+Overview
+********
+
+The Renesas RZ/G3S SMARC Evaluation Board Kit (RZ/G3S-EVKIT) consists of a SMARC v2.1 module board and a carrier board.
+
+* Device: RZ/G3S R9A08G045S33GBG
+
+  * Cortex-A55 Single, Cortex-M33 x 2
+  * BGA 359-pin, 14mmSq body, 0.5mm pitch
+
+* SMARC v2.1 Module Board Functions
+
+  * LPDDR4 SDRAM: 1GB x 1pc
+  * QSPI flash memory: 128Mb x 1pc
+  * eMMC memory: 64GB x 1pc
+  * PMIC power supply RAA215300A2GNP#HA3 implemented
+  * microSD card x2
+  * I3C connector
+  * JTAG connector
+  * ADC x8 channels
+  * Current monitor (USB Micro B)
+
+* Carrier Board Functions
+
+  * Gigabit Ethernet x2
+  * USB2.0 x2ch (OTG x1ch, Host x1ch)
+  * CAN-FD x2
+  * microSD card x1
+  * Mono speaker, Stereo headphone, Mic., and Aux..
+  * PMOD x2
+  * USB-Type C for power input
+  * PCIe Gen2 4-lane slot (G3S supports only 1-lane)
+  * M.2 Key E
+  * M.2 Key B and SIM card
+  * Coin cell battery holder (3.0V support)
+
+Hardware
+********
+
+The Renesas RZ/G3S MPU documentation can be found at `RZ/G3S Group Website`_
+
+.. figure:: rzg3s_block_diagram.webp
```

Comment:
@binhnguyen2434 : The picture in hardware should be rzg3s_smarc.webp rather than a block diagram.

Reply:
Thank you for the feedback.
Zephyr doc template has updated, [rzg3s_smarc.webp](https://github.com/zephyrproject-rtos/zephyr/pull/82016/files#diff-d55f2e77245a47a9d69c1c61d21192d228e5b7e86070daca6b1d98003d399896) is realized by Zephyr system. No need to mention directly in document.

Reply:
Thanks @binhnguyen2434 for your feedback. I understood.